modules/editor/inserter/button.js

```diff
@@ -0,0 +1,14 @@
+import Inserter from './';
+
+const h = wp.element.createElement;
+
+const InserterButton = ( { opened, onClick } ) => {
+	return h( 'div', { className: 'inserter__button' },
+		h( 'a', { onClick },
+			h( 'span', { className: 'dashicons dashicons-plus' } )
```

Comment:
There's been a few comments about icons that I'll defer to @jasmussen as to how to proceed.